### PR TITLE
chore: migrate custom renderer codelab to using imperative language.

### PR DESCRIPTION
**Pull request by jubishop**

### CODE REVIEW

File: codelabs/custom_renderer/custom_renderer.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Feedback Link: https://github.com/google/blockly-samples/issues/new/choose
 
 ### What you'll build
 
-You will build and use four renderers:
+This codelab build and use four renderers:
```
Comment (BeksOmega) [inline codelabs/custom_renderer/custom_renderer.md]:
```suggestion
This codelab builds and uses four renderers:
```

I'm not actually sure if this is correct :P

**Review by BeksOmega — APPROVED** [review]

File: codelabs/custom_renderer/custom_renderer.md (hunk 1 above)
Comment (jubishop) [reply codelabs/custom_renderer/custom_renderer.md]: yeah depends on how you look at it :) i think it's arguably correct!

Comment (jubishop) [reply codelabs/custom_renderer/custom_renderer.md]: should be fixed now.  thanks!

**Comment by jubishop** [comment]
hmm did i cause this somehow?

**Comment by jubishop** [comment]
it appears to be plugins/field-angle/src/field_angle.ts

**Comment by BeksOmega** [comment]
We've decided to temporarily ignore linting on PRs, but not disable the check! so you are good =)

**Pull request merged**